src/components/Person.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+function Person({ person }) {
+  const partner = person.sex === 'm' ? "wife's" : "husband's";
```

Comment:
```suggestion
  const partner = person.sex === 'm' ? "wife" : "husband";
```
and use it later like 
``` 
{`My ${partner}'s name...
```
because the variable `person` logically should contain some noun